app/templates/submit.hbs

```diff
@@ -156,6 +166,18 @@
                                     </div>
                                 </div>
                                 <div class="row">
+                                    <div class="col-md-6">
+                                        <h4>{{if newNode 'Apply' 'Also apply'}} this license to my OSF Project.</h4>
+                                        {{#unless newNode}}
+                                            <p>Selecting a license for your preprint does not automatically apply the license to your OSF project.</p>
+                                        {{/unless}}
```

Comment:
Also, spec says edit form has different language, don't know if this is still applicable: Apply this license to my OSF Project. Updating the license for your preprint does not automatically update the license on your OSF project.  The flag to know if you are on edit a preprint or adding a preprint (as both pages use this controller), is `editMode`.